infra/charts/feast-spark/charts/feast-jobservice/values.yaml

```diff
@@ -137,3 +137,6 @@ podLabels: {}
 
 # secrets -- Arbitrary secrets to mount on the job service pod, on /etc/secrets/<secret name>
 secrets: []
+
+# configMaps -- Arbitrary config maps to be mounted on the job service pod, on /etc/configs/<config name>
+configMaps: []
```

Comment:
If I understand correctly, the config map creation for whitelisting feature table has to be done outside of the helm release? 
I think that's not ideal and might increase the jobservice deployment complexity. Ideally, user can specify something like

```
whitelistedFeatureTable:
- feature_a
- feature_b
- ...
```

And the config map creation, mounting, as well as setting the environment variable should be configured by the helm chart itself.